std/ascii.zig

```diff
@@ -204,15 +200,15 @@ pub fn isBlank(c: u8) bool {
 
 pub fn toUpper(c: u8) u8 {
     if (isLower(c)) {
-        return c - 0x20;
+        return c | ~0b00100000;
     } else {
         return c;
     }
 }
 
 pub fn toLower(c: u8) u8 {
     if (isUpper(c)) {
-        return c + 0x20;
+        return c &  0b00100000;
```

Comment:
is the double space here desired?

Reply:
doesn't matter, it will get picked up by zig fmt eventually. let's focus our energy on the code, not the style